.github/workflows/coverage.yml

```diff
@@ -2,9 +2,9 @@ name: test_coverage
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, pre-check ]
   pull_request:
-    branches: [ main ]
+    branches: [ main, pre-check ]
```

Comment:
Reminder for @abadger  or myself. We need to take this out before merging.

Reply:
I've opened a PR with this change: https://github.com/oamg/convert2rhel/pull/802  We can merge that to the pre-check branch once we open the PR to merge pre-check to main.